app/assets/stylesheets/common/communicarts.scss

```diff
@@ -80,7 +80,19 @@ table td span.tag-values {
   font-weight: normal;
 }
 
+td.amount {
```

Comment:
`c2`? or `base` or something? I am not really clear on the purpose of this file but based on current name I would assume it's kind of a scss junk drawer?